test.mjs

```diff
@@ -205,7 +197,7 @@ const runTests = (baseopts) => {
       const spy = sinon.spy(function addSpy() {});
       watcher.on(EV.ADD, spy);
       await delay();
-      await write(testPath, dateNow());
+      await fsp.writeFile(testPath, dateNow());
```

Comment:
I liked the alias more. Can you assign write = fsp.writeFile? Same for rm, read.

Reply:
Sure can do 👍